### PR TITLE
Add user project limit to the admin panel

### PR DESCRIPTION
**Pull request by maciaszczykm**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes https://github.com/kubermatic/dashboard/issues/2458

**Special notes for your reviewer**:

**Release note**:

```release-note
Added option to set project limit for users from the admin panel.
```

### CODE REVIEW

File: src/app/settings/admin/admin-settings.component.html [hunk 1]
```diff
@@ -168,6 +169,27 @@
...
         <km-settings-status [isSaved]="isEqual(settings.enableOIDCKubeconfig, apiSettings.enableOIDCKubeconfig)"></km-settings-status>
       </div>
+
+      <div fxLayout="row">
+        <div fxFlex="16%"
+             fxLayoutAlign=" center"
+             class="entry-label">
+          <span>User Projects Limit</span>
+          <div class="km-icon-info"
+               matTooltip="Set limit to zero to allow unlimited project creation for users. It does not affects administrators."></div>
```
Comment (kgroschoff) [inline src/app/settings/admin/admin-settings.component.html:179]: isn't it "affect" instead of "affects"? 🤔

Comment (maciaszczykm) [reply src/app/settings/admin/admin-settings.component.html:179]: It is :) Good spot.

**Comment by maciaszczykm** [comment]
PTAL

**Pull request merged**